src/Searchbar.vue

```diff
@@ -15,6 +15,10 @@ export default {
   },
   computed: {
     primitiveData() {
+      if (this.value.length < 2) {
+        return [];
```

Comment:
Maybe we should just return `undefined` here instead of an empty array, since empty array might be viewed as "no matches", but in this case we are just preventing short search keywords.